From b2840744e714502d0e98d9c1d97cf8ebf3f5169d Mon Sep 17 00:00:00 2001
From: althafvly <althafvly@gmail.com>
Date: Thu, 1 Feb 2024 20:26:18 +0530
Subject: [PATCH 118/157] Revert "arch: arm64: configs: xiaomi: Set
 localversion to lineageos"

This reverts commit 97b681efd6b604ba0ce7895ddf59970b8094ef6a.

Change-Id: I10ff9274f562be331f94e1e178f6dc087a0fef99
---
 arch/arm64/configs/vendor/xiaomi/sm8250-common.config | 3 ---
 1 file changed, 3 deletions(-)

diff --git a/arch/arm64/configs/vendor/xiaomi/sm8250-common.config b/arch/arm64/configs/vendor/xiaomi/sm8250-common.config
index bb8795ef5f8e..36519f8feca1 100644
--- a/arch/arm64/configs/vendor/xiaomi/sm8250-common.config
+++ b/arch/arm64/configs/vendor/xiaomi/sm8250-common.config
@@ -1,6 +1,3 @@
-# Kernel
-CONFIG_LOCALVERSION="-lineageos"
-
 # DTBO
 CONFIG_BUILD_ARM64_DT_OVERLAY=y
 
-- 
2.48.1

